xwalk_android.gypi

```diff
@@ -22,6 +22,7 @@
       ],
       'include_dirs': [
         '..',
+        '../third_party/mojo/src',
```

Comment:
It depends on the build error you are getting when you don't add this, that is, take a look at which header(s) it complains about and try to figure out which target it comes from. Though to be honest I'd expect us to be using the mojo headers directly for this to fail like you describe, otherwise it's another target we depend on that is not correctly depending on mojo.